.github/CONTRIBUTING.md

```diff
@@ -249,7 +239,7 @@ git submodule update --init --recursive
 Note: before the next step make sure your docker has 4GB of memory ( Docker -> settings -> resources ) you should be able to manipulate the values here.
 
 cd docker
-docker compose -p lightdash-app -f docker-compose.dev.yml --env-file .env up --detach --remove-orphans 
+docker compose -p lightdash-app -f docker-compose.dev.yml --env-file .env up --detach --remove-orphans

```

Comment:
if we are inside ./docker will the command work ? as .env no longer exists in that folder ?

Reply:
updated docks